packages/atlas/src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -236,12 +240,33 @@ export const ChannelView: React.FC = () => {
     <ViewWrapper>
       <ChannelCover assetUrl={coverPhotoUrl} />
       <LimitedWidthContainer>
+        {smMatch ? (
+          <CollectorsBoxContainer>
+            <CollectorsBox
+              collectors={[
```

Comment:
If there are no collectors we will see: 
![image](https://user-images.githubusercontent.com/51168865/151392796-cd266a81-1f2e-41c0-83e0-08c6e5195f0b.png)

And if I remove CollectorsBox there will be a problem with spacing: 
![image](https://user-images.githubusercontent.com/51168865/151393199-bf62ed34-5726-4cb2-a0dc-775ff65ede2f.png)